bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -182,96 +193,206 @@ for tnum in template_ids:
                      'skipping' % tnum)
         continue
 
+    if type(rank_method.single_dtype) == list:
+        entries = [e[0] for e in rank_method.single_dtype]
+        if 'snr' in entries:
+            min_snr = min([numpy.min(sds_full[i]['snr']) for i in trigs.ifos])
+        else:
+            min_snr = None
+        if 'sigmasq'in entries:
+            max_sigmasq = max([numpy.max(sds_full[i]['sigmasq']) for i in trigs.ifos])
```

Comment:
This is slightly wrong in terms of actually getting the maximum possible network SNR (which is what this is used for later); you should be doing min() of the maxes in the list

Reply:
This has been fixed now